### ISSUE TITLE
Toogle analytics not working

### ISSUE THREAD

**Issue by andersonprante** [dauxio/daux.io#194]
**Describe the bug**
Toogle analytics not working by config.json. When config.json is defined with 
```json
"html": {
    "google_analytics": false,
    "piwik_analytics": false,
}
```
nothing happens. The analytics continue be present on compiled static files

**To Reproduce**
1. Set config.json with above settings
2. Run daux generate again
3. As inspect html files generated
4. As you can see, analytics scripts are present

**Expected behavior**
The expectation is the config will work

**Desktop (please complete the following information):**
 - daux/daux.io 0.12.1
 - Composer version 1.9.3 2020-02-04 12:58:49

**Comment by Davegb** [comment]
I'm having an issue too with Analytics, which makes daux take a lot of time to load on Chrome. I'd like to disable it as well.

**Comment by onigoetz** [comment]
The latest changes were released in version [0.13.0](https://github.com/dauxio/daux.io/releases/tag/0.13.0)
